test/tests/bootstrap/test_api20_os_bootstrap_parallel_local.py

```diff
@@ -0,0 +1,292 @@
+            },
+            {
+                "kvm": false,
+                "path": "/repo/centos/7.0",
+                "version": "7.0",
+                "workflow": "Graph.InstallCentOS"
+            },
+            {
+                "kvm": false,
+                "path": "/repo/rhel/7.0",
+                "version": "7.0",
+                "workflow": "Graph.InstallRHEL"
+            },
+            {
+                "kvm": false,
+                "path": "/repo/suse/42.1",
+                "version": "42.1",
+                "workflow": "Graph.InstallSUSE"
+            },
+            {
+                "kvm": false,
+                "path": "/repo/ubuntu",
+                "version": "trusty",
+                "workflow": "Graph.InstallUbuntu"
+            },
+            {
+                "kvm": false,
+                "path": "/repo/coreos",
+                "version": "899.17.0",
+                "workflow": "Graph.InstallCoreOS"
+            },
+            {
+                "kvm": true,
+                "path": "/repo/rhel/7.0",
+                "version": "7.0",
+                "workflow": "Graph.InstallRHEL"
+            },
+            {
+                "kvm": true,
+                "path": "/repo/centos/6.5",
+                "version": "6.5",
+                "workflow": "Graph.InstallCentOS"
+            },
+            {
+                "kvm": false,
+                "path": "/repo/winpe",
+                "productkey": "XXXXX-XXXXX-XXXXX-XXXXX-XXXXX",
+                "smbPassword": "onrack",
+                "smbRepo": "\\windowsServer2012",
+                "smbUser": "onrack",
+                "version": "2012",
+                "workflow": "Graph.InstallWindowsServer"
+            }
+        ],
+
+The OS repos are to be installed under 'on-http/static/http' directory reflecting the paths above.
+These can be files, links, or nfs mounts to remote repos in the following dirs:
+
+on-http/static/http/windowsServer2012 -- requires Samba share on RackHD server
+on-http/static/http/repo/centos/6.5
+on-http/static/http/repo/centos/7.0
+on-http/static/http/repo/rhel/7.0
+on-http/static/http/repo/suse/42.1
+on-http/static/http/repo/esxi/5.5
+on-http/static/http/repo/esxi/6.0
+on-http/static/http/repo/winpe
+on-http/static/http/repo/coreos/899.17.0
+
+'''
+
+import fit_path  # NOQA: unused import
+from nose.plugins.attrib import attr
+import fit_common
+import flogging
+log = flogging.get_loggers()
+
+# This gets the list of nodes
+NODECATALOG = fit_common.node_select()
+
+# dict containing bootstrap workflow IDs and states
+NODE_STATUS = {}
+
+# collect repo information from config files
+OSLIST = fit_common.fitcfg()["install-config"]["os-install"]
+
+# download RackHD config from host
+rackhdconfig = fit_common.rackhdapi('/api/2.0/config')['json']
+rackhdhost = "http://" + str(rackhdconfig['apiServerAddress']) + ":" + str(rackhdconfig['apiServerPort'])
+
+
+# this routine polls a task ID for completion
```

Comment:
In rackhd namespace, the `task` has special meaning. But at here, the `task` is actually 'workflow', so to avoid confusion, I'd suggest changing the word from `task` to `workflow`. The same for whole rest of code.